libbeat/dashboards/modify_json.go

```diff
@@ -46,25 +46,34 @@ func ReplaceIndexInIndexPattern(index string, content common.MapStr) common.MapS
 		return content
 	}
 
-	objects, ok := content["objects"].([]interface{})
-	if !ok {
-		return content
-	}
-
-	// change index pattern name
-	for i, object := range objects {
-		objectMap, ok := object.(map[string]interface{})
-		if !ok {
-			continue
+	objectMaps, ok := content["objects"].([]common.MapStr)
```

Comment:
You might as well constrain the scope of `objectMaps` to the `if` block by declaring as part of the if statement just like it's done in the `if else`.